arm-image-installer

```diff
@@ -606,6 +632,34 @@ if [ "$SSH_KEY" != "" ]; then
 		echo "= WARNING: No SSH Key Added."
 	fi
 fi
+# Configure Wi-Fi if SSID and password are provided
+if [ "$WIFI_SSID" != "" ] && [ "$WIFI_PASS" != "" ]; then
+    echo "= Configuring Wi-Fi: $WIFI_SSID"
+    mkdir -p ${PREFIX}/etc/NetworkManager/system-connections/
+    # Use the SSID as the filename (replace spaces and special characters with underscores)
+    FILENAME=$(echo "$WIFI_SSID" | tr ' ' '_' | tr -cd '[:alnum:]._-').nmconnection
+    cat > ${PREFIX}/etc/NetworkManager/system-connections/$FILENAME <<-EOF
+[connection]
+id=$WIFI_SSID
+type=wifi
+interface-name=wlan0
+
+[wifi]
+mode=infrastructure
+ssid=$WIFI_SSID
+
+[wifi-security]
+key-mgmt=wpa-psk
```

Comment:
We can't assume that, it depends on the AP and with WPA3 it's possibly sae

Reply:
Thanks for the feed back , I parameterized it for open,wap2 and wpa3.
Did more research and found out that there enterprise grade key management like: wpa-eap or wpa-eap-suite-b-192, which I intentionally left that out  to keep it simple and avoiding additional flags in the installer. My intention is primarily provide hobbyist with simple network setup similar to raspberry pi imager.

Reply:
Yes, enterprise is complex, and at times you need to deal with 2FA or even client side certs. Maybe add a note in the help or in the README